code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -102,6 +102,9 @@ GLOBAL_LIST_EMPTY(features_by_species)
 	///Replaces default appendix with a different organ.
 	var/obj/item/organ/internal/appendix/mutantappendix = /obj/item/organ/internal/appendix
 
+	///Multiplier for the race's speed. Positive numbers make it move slower, negative numbers make it move faster.
+	var/speedmod = 0
```

Comment:
If we're reverting this JUST for snails then why not JUST make it handled by `/datum/species/snail` than slapping it back on `/datum/species`?